.github/actions/setup-playwright/action.yml

```diff
@@ -0,0 +1,30 @@
+name: Setup Python + Playwright
+description: Common setup for all Playwright-based smoke tests
+inputs:
+  python-version:
+    description: Version of Python to use
+    required: false
+    default: "3.11"
+  exclude_inputs:
+    description: Comma-separated list of inputs to exclude from masking
+    required: false
+    default: target_url,course_id
+
+runs:
+  using: composite
+  steps:
+    - name: Set up Python
```

Comment:
Could we add here a cache step like this.
https://github.com/nelc/eox-nelp/blob/76fdc2245533faa2a5f6f34fdd5476854dc706f4/.github/workflows/tests.yml#L22-L32

Also we could add a integration requirements file to add eg `pytest-playwright`. And try to dont reconfigure python and the pip requirement for each test.